ui/e2e/src/fixtures/dashboardTest.ts

```diff
@@ -28,6 +31,13 @@ type DashboardTestFixtures = {
    * tests because it will unnecessarily slow them down.
    */
   modifiesDashboard: boolean;
+
+  /**
+   * Time in unix milliseconds to be returned when `Date.now` is called on the
+   * page. Useful for stabilizing tests that depend on the current time.
+   */
+  mockNow: number;
```

Comment:
Is it simpler if we make this optional and default it to `0`?

Reply:
It gets made optional at the point we need to use it in tests like the example below. Something deeper in the `testBase.extend` generic type transforms it. It's defaulted to `0` in the initial setup of the fixture.

```js
test.use({
  dashboardName: 'Panels',
});
```

Reply:
In the `0` default case, the date is not mocked. I'm inclined to avoid mocking it all the time unless we really need it for the majority of our tests.